cli/src/api/rename.ts

```diff
@@ -22,8 +22,14 @@ export async function renameProject(userOptions: RenameOptions) {
   const packageJsonData = JSON.parse(packageJsonContent)
 
   merge(
-    packageJsonData,
-    omitBy(pick(options, ['name', 'description', 'author', 'license']), isNil),
+    merge(
+      packageJsonData,
+      // @ts-expect-error missing fields: author and license
+      omitBy(
+        pick(options, ['name', 'description', 'author', 'license']),
```

Comment:
`es-toolkit` is more strict, and the `author` and `license` fields are missing in `options`.